pepys_import/resolvers/command_line_resolver.py

```diff
@@ -1,264 +1,701 @@
         else:
-            chosen_sensor_type = sensor_types[choice - 2]
+            chosen_datafile_type = self.fuzzy_search_datafile_type(data_store)
 
-        return sensor_name, chosen_sensor_type
+        # Choose Privacy
+        if privacy:
+            chosen_privacy = data_store.add_to_privacies(privacy)
+        else:
+            chosen_privacy = self.resolve_privacy(data_store)
 
-    def resolve_sensor(self, data_store, sensor_name):
-        choice = get_choice_input(
-            f"Sensor '{sensor_name}' not found. Do you wish to: ",
-            [f"Add a new sensor, titled '{sensor_name}'", "Cancel import"],
+        print("Input complete. About to create this datafile:")
+        print(f"Name: {datafile_name}")
+        print(f"Type: {chosen_datafile_type.name}")
+        print(f"Classification: {chosen_privacy.name}")
+
+        choice = create_menu(
+            "Create this datafile?: ", ["Yes", "No, make further edits"]
         )
 
-        if choice == 1:
-            return self.add_to_sensors(data_store, sensor_name)
-        elif choice == 2:
+        if choice == str(1):
+            return datafile_name, chosen_datafile_type, chosen_privacy
+        elif choice == str(2):
+            return self.add_to_datafiles(data_store, datafile_name, None, None)
+        elif choice == ".":
             print("Quitting")
             sys.exit(1)
 
-    def resolve_privacy(self, data_store, table_type_id, table_name):
-        if table_type_id in self.table_privacies:
-            return table_type_id, self.table_privacies[table_type_id]
+    def add_to_platforms(
+        self, data_store, platform_name, platform_type, nationality, privacy
+    ):
+        """
+        This method resolves platform type, nationality and privacy. It asks user
+        whether to create a platform with resolved values or not. If user enters Yes,
+        it returns all necessary data to create a platform.
+        If user enters No, it resolves values again.
+
+        :param data_store: A :class:`DataStore` object
+        :type data_store: :class:`DataStore`
+        :param platform_name: Name of :class:`Platform`
+        :type platform_name: String
+        :param nationality: Name of :class:`Nationality`
+        :type nationality: Nationality
+        :param platform_type: Name of :class:`PlatformType`
+        :type platform_type: PlatformType
+        :param privacy: Name of :class:`Privacy`
+        :type privacy: Privacy
+        :return:
+        """
+        print("Ok, adding new platform.")
+
+        platform_name = prompt("Please enter a name: ", default=platform_name)
+        trigraph = prompt(
+            "Please enter trigraph (optional): ", default=platform_name[:3]
+        )
+        quadgraph = prompt(
+            "Please enter quadgraph (optional): ", default=platform_name[:4]
+        )
+        pennat_number = prompt("Please enter pennat number (optional): ", default="")
```

Comment:
minor typo @BarisSari  - it's `pennant`, not `pennat`

https://en.wikipedia.org/wiki/Pennant_number

Reply:
Corrected:

https://github.com/debrief/pepys-import/pull/73/commits/d4b20701eb383aaebfd6eab5d83cfa70c7259f06